app/shared/profile-commands.js

```diff
@@ -58,8 +58,11 @@ export function closeBrowserWindow(): ProfileCommand {
   };
 }
 
-export function newBrowserWindow(): ProfileCommand {
+export function newBrowserWindow(tabInfo: ?Object): ProfileCommand {
```

Comment:
We don't, and it's all currently disabled.  Let's address this when we re-enable tab dragging and dropping.